File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/DataViz/Scatter_Charts/Multiple_AxesController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Scatter_ChartsController : Controller
    {
        public ActionResult Multiple_Axes()
        {
            return View(ChartDataRepository.EngineData());
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/scattercharts/MultipleAxesController.java
package com.kendoui.spring.controllers.scattercharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-scatter_charts-multiple_axes-controller")
@RequestMapping(value="/scatter-charts/")
public class MultipleAxesController {
    @RequestMapping(value = "/multiple-axes", method = RequestMethod.GET)
    public String index(Model model) {
        model.addAttribute("engineData", ChartDataRepository.EngineData());
        
        return "/scatter-charts/multiple-axes";
    }
}
